br/pkg/task/backup.go

```diff
@@ -45,11 +48,13 @@ const (
 	flagRemoveSchedulers = "remove-schedulers"
 	flagIgnoreStats      = "ignore-stats"
 	flagUseBackupMetaV2  = "use-backupmeta-v2"
+	flagNoCheckpoint     = "no-checkpoint"
```

Comment:
maybe rename it to flagUseCheckpoint and set default value to true.

Reply:
don't let user use double negative grammer

Reply:
done